skopt/gp_opt.py

```diff
@@ -95,7 +96,9 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         Number of calls to `func`.
         If `n_random_starts` > 0, `n_calls - n_random_starts`
         additional evaluations of `func` are made that are guided
-        by the `base_estimator`.
+        by the `base_estimator`. If `x0` is provided but not `y0`
+        then `n_calls - len(x0) - n_random_starts` evaluations
+        are made instead of `n_calls - n_random_starts` .
```

Comment:
Can we move this text further up in the doc string? It relates to both `x0`, `n_calls`, `n_random_starts`, etc and is about the whole function not just the `n_calls` parameter.

For `n_calls` I propose: Maximum number of calls to `func`.